lte/gateway/fabfile.py

```diff
@@ -532,33 +532,29 @@ def run_with_retry(c_gw, command, retries=10):
 
 
 @task
-def get_test_summaries(
-        c,
-        dst_path="/tmp",
-        integration_tests=False,
-        sudo_tests=False,
-        dev_vm_name="magma",
-):
-    c.run('mkdir -p ' + dst_path)
+def get_integ_test_summaries(c):
+    _get_test_summaries_from_vm(c, "magma_test")
```

Comment:
Hard-coding `dst_path` makes sense. Having two functions that only differ by `vm_name` kind of defeats the point of having it as a variable - this moves the variable input into the function name. I would prefer to keep `vm_name` as a function input.

Reply:
I would not have `vm_name` as an input, but rather the meaning of the function call. Maybe something like
`fab get-test-summaries --sudo-tests`/`fab get-test-summaries --integ-tests` similar to what is is on master? This would move the knowledge on which machine you have to look for the test results into a secondary step.